content/news/2022-04-12-bevy-0.7/index.md

```diff
@@ -0,0 +1,654 @@
+[`AnimationClip`]: https://docs.rs/bevy/0.7.0/bevy/animation/struct.AnimationClip.html
+[`Transform`]: https://docs.rs/bevy/0.7.0/bevy/transform/components/struct.Transform.html
+
+## GLTF Animation Importing
+
+<div class="release-feature-authors">authors: @mockersf</div>
+
+Bevy's GLTF importer was extended to import GLTF animations into the new [`AnimationPlayer`] system. This supports both "skeletal animation" and arbitrary transform animations:
+
+<video controls loop><source  src="fox.mp4" type="video/mp4"/></video>
+
+```rust
+struct FoxAnimations {
+    walk: Handle<AnimationClip>,
+}
+
+fn setup(mut commands: Commands) {
+    commands.spawn_scene(asset_server.load("models/animated/Fox.glb#Scene0"));
+    commands.insert_resource(FoxAnimations {
+        walk: asset_server.load("models/animated/Fox.glb#Animation0"),
+    });
+}
+
+fn play_on_load(
+    animations: Res<FoxAnimations>,
+    mut players: Query<&mut AnimationPlayer, Added<AnimationPlayer>>,
+) {
+    for mut player in players.iter_mut() {
+        player.play(animations.walk.clone()).repeat();
+    }
+}
+```
+
+## Unlimited* Point Lights
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump), @robtfm</div>
+
+Bevy can now render scenes with arbitrary numbers of point lights on platforms that support storage buffers (which is basically everything but WebGL). In the last Bevy release (0.6) we added [Clustered Forward Rendering](/news/bevy-0-6/#clustered-forward-rendering), which is a rendering technique that optimizes each fragment's light calculation costs by assigning lights to "clusters". However in the interest of platform compatibility (WebGL), we initially limited ourselves to 256 lights because that is what fit in a uniform buffer binding.
+
+In **Bevy 0.7**, we added the ability to automatically "upgrade" to using unbounded storage buffers for Clustered Forward Rendering on platforms that support them, enabling unlimited* point lights. There is an asterisk there because in practice this is limited by memory and hardware constraints. 
+
+## Light Clustering Features and Optimizations
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump), @robtfm, @dataphract, @cart</div>
+
+With the upper limit of 256 point lights removed, the only limit on lights is what the hardware can support and bottlenecks in our algorithms. To increase the number of lights, we made a number of optimizations to our clustering algorithms.
+
+* **Dynamic Light Clusters**: By default cluster x/y slices are now dynamically configured based on the lights in the scene, which can significantly increase performance in some scenes. The clustering behavior is now also user-configurable as FixedZ (the new default dynamic x/y behavior), custom fixed x/y/z slice values, single-cluster, and "no clustering".
+* **Light Frustum Change Detection**: We now use Bevy ECS's change detection feature to only recalculate the view frustum of lights that have changed.
```

Comment:
This feels minor to me. It’s not a heavy calculation.

Reply:
Oh, the light view frustum! For some reason when I read this the first time, I thought it meant the camera view frustum.

Reply:
I can't resolve this so I'll just delete it.